Sources/MapboxCoreNavigation/AlternativeRoute.swift

```diff
@@ -0,0 +1,88 @@
+import Foundation
+import MapboxDirections
+import MapboxNavigationNative
+import Turf
+
+/**
+ Representation of an alternative route with relation to the original.
+ 
+ This struct contains main and alternative routes which are sharing same origin and destination points, but differ at some point.
+ */
+public struct AlternativeRoute: Identifiable {
+    /// Alternative route identificator type
+    public typealias ID = UInt32
+    /// Breif statistics of a route for travelling
+    public struct RouteInfo {
```

Comment:
This struct is just to contain a pair of values, having a Route object here is misleading because it is not supposed to contain an entire route information and there are `RouteInfo`'s for the entire alternative route and for portion after the deviation point. The former is not supposed to be treated as individual Route.
To get an actual Route object, user can use `indexedRouteResponse` instead. At this point having a route and `infoFromOrigin` might be a bit redundant since they share the same info, but I find it more convenient to use and understand that way.